airbyte-integrations/base-singer/base_singer/singer_helpers.py

```diff
@@ -109,7 +140,7 @@ def create_singer_catalog_with_selection(masked_airbyte_catalog, discovered_sing
                     new_metadata = metadata
                     new_metadata["metadata"]["selected"] = True
                     if not is_field_metadata(new_metadata):
-                        new_metadata["metadata"]["forced-replication-method"] = "FULL_TABLE"
+                        new_metadata["metadata"]["replication-method"] = "FULL_TABLE"
```

Comment:
How does this react with schemas that do set `forced-replication-method` like Stripe?

Reply:
forced-replication-method was not respected by postgres tap, so I am now setting both.